packages/components/phone-number/src/components/osds-phone-number/interfaces/attributes.ts

```diff
@@ -1,11 +1,11 @@
 import type { ODS_COUNTRY_ISO_CODE, ODS_LOCALE } from "@ovhcloud/ods-common-core";
-import { ODS_PHONE_NUMBER_COUTRIE } from "../constants/phone-number-countries";
+import { ODS_PHONE_NUMBER_COUNTRY_PRESET } from "../constants/phone-number-countries";
 
 interface OdsPhoneNumberAttribute {
   /** Ability to clear the phone number value */
   clearable?: boolean;
   /** A specific subset of countries to display in the select instead of the whole list  */
-  countries?: ODS_COUNTRY_ISO_CODE[] | ODS_PHONE_NUMBER_COUTRIE.All;
+  countries?: ODS_COUNTRY_ISO_CODE[] | ODS_PHONE_NUMBER_COUNTRY_PRESET.All | string;
```

Comment:
should be `countries?: ODS_COUNTRY_ISO_CODE[] | ODS_PHONE_NUMBER_COUNTRY_PRESET| string;`
Even if we have only All for now, it should accept any value of it